packages/edit-site/src/components/page-templates/style.scss

```diff
@@ -51,8 +51,17 @@
 	}
 }
 
+.dataviews-view-table {
+	.page-templates-description {
+		margin-bottom: $grid-unit-10;
```

Comment:
If we do this then the margin will also be applied in Grid and List layouts, which we do not want.

Reply:
Ah, that's true. How about the following? It may be a matter of preference.

```scss
.page-templates-description {
	max-width: 50em;
	text-wrap: balance; // Fallback for Safari
	text-wrap: pretty;

	.dataviews-view-table & {
		margin-bottom: $grid-unit-10;
	}
}
```

Reply:
Oh sure, I always forget about that selector.